### PR TITLE
Fix acceptance tests, and minor doc tweaks

### PR DESCRIPTION
**Pull request by andy-armstrong**
This commit fixes a few issues from my previous ORA PR (https://github.com/edx/edx-ora2/pull/747):
* Updated some doc strings based on feedback from @catong 
* Fixed the acceptance tests which I broke when I introduced DDT (I was running the wrong tests)
* Fixed the Jenkins build to have a DDT dependency
* Renamed the environment variable to ORA_SANDBOX_URL so as to be more explicit

@muzaffaryousaf @cahrens @catong please review.

### CODE REVIEW

File: openassessment/templates/openassessmentblock/staff_area/staff_area.html [hunk 1]
```diff
@@ -103,7 +103,7 @@ <h2 class="staff-info__title ui-staff__title">
                 </div>
 
                 <div class="staff-info__status ui-staff__content__section">
-                    <table class="staff-info__status__table" summary="{% trans "Where are your learners currently in this problem" %}">
```
Comment (cahrens) [inline openassessment/templates/openassessmentblock/staff_area/staff_area.html]: Was this string supposed to move someplace else?

File: test/acceptance/auto_auth.py [hunk 2]
```diff
@@ -2,12 +2,13 @@
...
 
+import os
 import re
 import urllib
+
 from bok_choy.page_object import PageObject
 
-import os
-BASE_URL = os.environ.get('BASE_URL')
+ORA_SANDBOX_URL = os.environ.get('ORA_SANDBOX_URL', failobj=os.environ.get('BASE_URL'))
```
Comment (cahrens) [inline test/acceptance/auto_auth.py]: Is this necessary even with the backward compatibility code in test-acceptance.sh?

File: test/acceptance/pages.py [hunk 3]
```diff
@@ -1,11 +1,13 @@
...
 """
+
+import os
+
 from bok_choy.page_object import PageObject
 from bok_choy.promise import EmptyPromise
 
-import os
-BASE_URL = os.environ.get('BASE_URL')
+ORA_SANDBOX_URL = os.environ.get('ORA_SANDBOX_URL', failobj=os.environ.get('BASE_URL'))
```
Comment (cahrens) [inline test/acceptance/pages.py]: Same question.

**Comment by cahrens** [comment]
Just a few questions, otherwise :+1:

File: openassessment/templates/openassessmentblock/staff_area/staff_area.html (hunk 1 above)
Comment (andy-armstrong) [inline openassessment/templates/openassessmentblock/staff_area/staff_area.html]: Reviewers: @catong thought this summary was confusing, and in our investigations it appears that the attribute has been deprecated anyway:

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/table

**Comment by catong** [comment]
:+1:

**Comment by muzaffaryousaf** [comment]
:+1:  Thanks for the import fix.

File: openassessment/templates/openassessmentblock/staff_area/staff_area.html (hunk 1 above)
Comment (andy-armstrong) [inline openassessment/templates/openassessmentblock/staff_area/staff_area.html]: @cahrens You beat me to my reviewer comment... see above.

File: test/acceptance/auto_auth.py (hunk 2 above)
Comment (andy-armstrong) [inline test/acceptance/auto_auth.py]: I was nervous that it wouldn't work but thinking about it some more it seems that it should. I'll take it out and then only add it back in if the tests still fail.

**Pull request merged**